Cloudinary/Classes/Core/Features/Helpers/CLDTransformation.swift

```diff
@@ -23,6 +23,7 @@
 //
 
 import Foundation
+import CoreGraphics
```

Comment:
What is this import? If on other change was done in the code?

Reply:
we need this import as we use CGPoint, it was missing in the branch despite adding it to the previous PR